virttest/shared/cfg/base.cfg

```diff
@@ -578,9 +578,16 @@ env_cleanup = no
 
 # Verify host dmesg in postprocess.
 verify_host_dmesg = yes
+# Comma separated list of regular expressions enclosed with "'" whose
+# matches are to be ignored during verification, for example: 'x.*', 'y.*'
```

Comment:
perhaps, it sound more straightforward that separated list is  enclosed with [], e.g ['x.*',  'y.*']

Reply:
But that's not how it's used, a sample is really. In the sample test run you can see that I've used
```
--vt-extra-params "expected_guest_dmesg = \"'.*this driver is not recommended for new deployments.*'\""
```
That is, in a configuration file it would look something like:
```
expected_guest_dmesg = '.*this driver is not recommended for new deployments.*', '.*some other expected message.*'
```